docker/stand-alone/.env

```diff
@@ -0,0 +1,13 @@
+# https://docs.docker.com/compose/env-file/
+# https://docs.docker.com/compose/environment-variables/#the-env-file
+
+# These values MUST be changed in order to be able to authenticate with SAM
+GOOGLE_OAUTH_CLIENT_ID=change_me
+GOOGLE_CLOUD_PROJECT_DOMAIN=change_me
+
+# Variables below can optionally be modified as needed
+SAM_APP_IMAGE=broadinstitute/sam:stand-alone
+SAM_HOST=localhost
+EMAIL_DOMAIN=example.foo
```

Comment:
What is this used for and what should the value be? Looks like it's optional so it may not matter, but it would be nice to understand.